.github/scripts/verifyPodfile.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 
-declare -r GREEN='\033[0;32m'
+# This script accepts two arguments:
+# First argument (required): the branch to compare against for removals
+# Second argument (optional): `-v` which will output each branch's podspecs and the removals if they exist
+
 declare -r RED='\033[0;31m'
```

Comment:
It would be great to use the utilities from the [shellUtils](https://github.com/Expensify/App/blob/main/scripts/shellUtils.sh) file here to reduce duplication:

```bash
START_DIR=$(pwd)
ROOT_DIR=$(dirname "$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")")
cd "$ROOT_DIR" || exit 1
source scripts/shellUtils.sh

...
...

# Then add the end of the script go back to the starting dir
cd "$START_DIR" || exit 1
```

Then we can remove the color variables from this file and make some replacements like:

```diff
- echo -e "${GREEN}Podfile checksum verified!${NC}"
+ success 'Podfile checksum verified!'
...
...
- echo -e "${RED}Error: Podfile.lock checksum mismatch. Did you forget to run \`npx pod-install\`?${NC}"
+ error 'Error: Podfile.lock checksum mismatch. Did you forget to run \`npx pod-install\`?'
```

Reply:
We may be able to get away with `source ./scripts/shellUtils.sh` here as is done in the [validateActionsAndWorkflows](https://github.com/Expensify/App/blob/main/.github/scripts/validateActionsAndWorkflows.sh) script.  The Github runner invokes the script from the project root.  I may be missing something here so I added the guards in my latest commit. The script has multiple exit points so I refactored the exit procedure into it's own function.

Reply:
It actually looks like the runners don't have the realpath command as seen here: https://github.com/redstar504/test-gh-workflows/actions/runs/3832780888/jobs/6523488009

Reply:
I was able to implement this in my latest revision. It was useful for checking the existence of the directories from the Podfile.lock, as they were outputted as relative to the ios directory.  I was unable to use the realpath command as it's not included on the runner, but the same idea seems to work with a relative path.

Reply:
Yeah, the only reason to use `realpath` would be so that the script can be run from anywhere and work the same, but I often forget that `realpath` isn't completely portable :frowning_face: